MAINTAINERS.yml

```diff
@@ -3384,7 +3384,7 @@ ITE Platforms:
     - drivers/*/*it8xxx2*.c
     - dts/bindings/*/*ite*
```

Comment:
While you're at it could you make this one more specific if possible, please? It currently matches "display/ilitek..."

Reply:
I'll open a follow-up PR with more risc-v enhancements